packages/web-react/src/types/pill.ts

```diff
@@ -1,7 +1,10 @@
 import { ComponentPropsWithRef, ElementType } from 'react';
-import { ChildrenProps, EmotionColorsDictionaryType, StyleProps, TransferProps } from './shared';
+import { BasicPillColors } from '../components/Pill';
```

Comment:
```suggestion
import { BasicPillColors } from '../components';
```

should be enough and is same as in tag